static/js/compose_actions.js

```diff
@@ -62,19 +62,35 @@ function get_focus_area(msg_type, opts) {
     }
```

Comment:
I wasn't totally sure what belonged in `compose_actions` and not `compose_ui` but I took my best guess

Reply:
`compose_ui` is primarily for clicking buttons in the compose box -- most of the code is around mutating the compose box contents when clicking things. It would be a great preparatory project to rename that module to `compose_textarea_ui.js`, with prep commits to move the few functions that don't belong.

I think in `main`, `compose_ui.js` just has a few things that don't belong: `hide_compose_spinner` and `show_compose_spinner`, with their state variable, which perhaps belong in `compose_state.js`, and `compose_ui.get_compose_click_target`, which probably could be moved into a new `compose_popovers.js` (with other code to migrate there later), or maybe `compose_actions.js` if that doesn't cause import cycles.

@N-Shar-ma FYI -- I think it'd be great if either you or @evykassirer does this rename, as I think it'd make this part of the codebase considerably more readable.

(I think `compose.js` vs. `compose_actions.js` will still have a similar problem after this, but disentangling `compose_textarea_ui.js` will help a lot).

Reply:
Started thread [here](https://chat.zulip.org/#narrow/stream/6-frontend/topic/compose.20file.20names/near/1492637)